docs/source/test_network.md

```diff
@@ -0,0 +1,578 @@
+test network, each organization operates one peer each, `peer0.org1.example.com`
+and `peer0.org2.example.com`.
+
+Every Fabric network also includes an ordering service that is the common biding
+of the network. While peers validate transactions and add blocks of transactions
+to the blockchain ledger, they do not decide on the order of transactions or
+include them into new blocks. On a distributed network, peers may be running far
+away from each other and not have a common view of when a transaction was created.
+Coming to consensus on the order of transactions is a costly process that would
+create overhead for the peers.
+
+An ordering service allows peers to focus on validating transactions and
+committing them to the ledger. After they receive transactions from peers,
+ordering nodes come to consensus on the order of transactions and then add them
+to blocks. The blocks are then distributed to peer nodes, which then add the blocks
+the blockchain ledger. Ordering nodes also operate the system channel that
+defines the capabilities of a Fabric network, such as how blocks are made and
+which version of Fabric that nodes can use. The system channel defines which
+organizations are members of the consortium.
+
+The sample network uses a single node Raft ordering service that is operated by
+the ordering organization. You can see the ordering node running on your machine
+as `orderer.example.com`. While the test network only uses a single node ordering
+service, a real network would have multiple ordering nodes, operated by one or
+multiple orderer organizations. The different ordering nodes would use the Raft
+consensus algorithm to come to agreement on the order of transactions across
+the network.
+
+## Creating a channel
+
+Now that we have peer and orderer nodes running on our machine, we can use the
+nodes to create a Fabric channel. Channels are a private layer of communication
+that can be used by only certain network members. Channels can be used only by
+organizations that are invited the channel, and are invisible to other members
+of the network. Each channel has separate blockchain ledger. Organizations that
+have been invited can "join" their peers to the channel to store the channel
+ledger and validate the transactions on the channel.
+
+You can use the `network.sh` script to create a channel for `org1.example.com`
+and `org2.example.com` and join their peers to the channel. Run the following
+command to create a channel called `mychannel`:
+```
+./network.sh createChannel
+```
+If the command was successful, you can see the following message printed in your
+logs:
+```
+========= Channel successfully joined ===========
+```
+
+You can create additional channels by using the same `createChannel` command and
+using the channel flag to specify a new channel name. As an example, you can use
+the following command to create a second channel named `channel2`:
+```
+./network.sh createChannel -c channel2
+```
+
```

Comment:
Could I use this command first, before creating mychannel?

Reply:
Yup, changing this section in response to similar feedback from Pam.